package.json

```diff
@@ -56,10 +56,10 @@
   },
   "devDependencies": {
     "difflet": "~0.2.3",
-    "grunt-contrib-jshint": "~1.0.0",
+    "eslint-config-grunt": "^1.0.1",
     "grunt-contrib-nodeunit": "~0.4.1",
     "grunt-contrib-watch": "~1.0.0",
-    "grunt-jscs": "~2.8.0",
+    "grunt-eslint": "^18.0.0",
```

Comment:
`19.0.0` is not used since it doesn't supports node < 4, which means you can't use ESLint 3.0, we can apply same approach as in jquery repo though - https://github.com/jquery/jquery/commit/030191ae32dfcb7ecb2efb61d17a4964a3633e44